apps/daimo-bot/api/trpcClient.ts

```diff
@@ -0,0 +1,40 @@
+import type { AppRouter } from "@daimo/api";
+import { createTRPCProxyClient, httpBatchLink } from "@trpc/client";
+
+const BASE_CHAIN_ID = "8453";
+const BASE_SEPOLIA_CHAIN_ID = "84532";
+const URL_CHAIN_MAPPING = {
+  mainnet: {
+    url: process.env.DAIMO_APP_API_URL_MAINNET,
+    chainId: BASE_CHAIN_ID,
+  },
+  testnet: {
+    url: process.env.DAIMO_APP_API_URL_TESTNET,
+    chainId: BASE_SEPOLIA_CHAIN_ID,
+  },
+  default: {
+    url: "http://localhost:3000",
+    chainId: BASE_SEPOLIA_CHAIN_ID,
+  },
+} as const;
```

Comment:
a small simplification you can do to replace URL_CHAIN_MAPPING:

below `const { url, chainId } = getEthApiConfig()`

```ts
function getEthApiConfig() {
  let url = process.env.DAIMO_APP_API_URL_MAINNET;
  if (url) return { url, chainId: BASE_CHAIN_ID };
  url = process.env.DAIMO_APP_API_URL_TESTNET || "http://localhos:3000";
  return { url, chainId: BASE_SEPOLIA_CHAIN_ID };
}
```